src/HashBus.Projector.TopTweetersRetweeters/TopTweetersRetweetersProjection.cs

```diff
@@ -19,12 +21,12 @@ public TopTweetersRetweetersProjection(IRepository<string, IEnumerable<TweetRetw
             this.tweets = tweets;
         }
 
-        public void Handle(Application.Events.TweetAnalyzed message)
+        public Task Handle(TweetAnalyzed message, IMessageHandlerContext context)
         {
             var trackTweets = this.tweets.GetAsync(message.Tweet.Track).GetAwaiter().GetResult().ToList();

```

Comment:
make the Handle method `async` and remove `.GetAwaiter().GetResult()`, making it:

```
( await this.tweets.GetAsync(message.Tweet.Track) ).ToList();
```

then you can remove all the `return Task.FromResult(0);`